cluster/addons/fluentd-gcp/fluentd-gcp-configmap-old.yaml

```diff
@@ -61,16 +61,18 @@ data:
       # reform.var.log.containers.<POD_NAME>_<NAMESPACE_NAME>_<CONTAINER_NAME>-<CONTAINER_ID>.log
       tag reform.*
       read_from_head true
-      format multi_format
```

Comment:
Why are you  changing a configmap for old resource model in a PR that adds support for both old and new model?